docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md

```diff
@@ -79,7 +79,7 @@ NOTE: the various other Cardano components can be obtained through other attribu
    - `cardano-explorer`, `cardano-explorer-swagger`, `cardano-explorer-mock`
 -  `cardano-sl-tools`:
    - `cardano-analyzer`, `cardano-dht-keygen`, `cardano-genupdate`, `cardano-keygen`, `cardano-launcher`, `cardano-addr-convert`, `cardano-cli-docs`, `cardano-block-gen`, `cardano-post-mortem`
--  `cardano-sl-wallet-static`:
+-  `cardano-sl-wallet-new-static`:
```

Comment:
This is a problem now when `cardano-sl-wallet-static` is removed.

Is it fine/correct in the context to replace with `cardano-sl-wallet-static` as I've done here?

Reply:
I suppose :no_mouth: